angular/src/directives/overlays/popover.ts

```diff
@@ -73,7 +73,7 @@ export declare interface IonPopover extends Components.IonPopover {
 @Component({
   selector: 'ion-popover',
   changeDetection: ChangeDetectionStrategy.OnPush,
-  template: `<ng-container [ngTemplateOutlet]="template" *ngIf="isCmpOpen"></ng-container>`,
+  template: `<div class="ion-page"><ng-container [ngTemplateOutlet]="template" *ngIf="isCmpOpen"></ng-container></div>`,
```

Comment:
Actually - the original changes only address `ion-modal`. I was unable to replicate the issue requiring adding `.ion-page` to popovers. That my be unnecessary here.